plugins/modules/database/postgresql/postgresql_user.py

```diff
@@ -39,33 +38,33 @@
     description:
     - Set the user's password, before 1.4 this was required.
     - Password can be passed unhashed or hashed (MD5-hashed).
-    - Unhashed password will automatically be hashed when saved into the
-      database if C(encrypted) parameter is set, otherwise it will be save in
+    - Unhashed password is automatically hashed when saved into the
+      database if I(encrypted) option is set, otherwise it's saved in
       plain text format.
-    - When passing an MD5-hashed password it must be generated with the format
+    - When passing an MD5-hashed password, you must generate it with the format
       C('str["md5"] + md5[ password + username ]'), resulting in a total of
       35 characters. An easy way to do this is C(echo "md5$(echo -n
       'verysecretpasswordJOE' | md5sum | awk '{print $1}')").
     - Note that if the provided password string is already in MD5-hashed
-      format, then it is used as-is, regardless of C(encrypted) parameter.
+      format, then it is used as-is, regardless of I(encrypted) option.
     type: str
   db:
     description:
-    - Name of database to connect to and where user's permissions will be granted.
+    - Name of database to connect to and where user's permissions is granted.
     type: str
     aliases:
     - login_db
   fail_on_user:
     description:
-    - If C(yes), fail when user (role) can't be removed. Otherwise just log and continue.
-    default: 'yes'
+    - If C(yes), fails when cannot remove the user (role). Otherwise just log and continue.
```

Comment:
This is grammatically wrong. Maybe "If C(yes), fails when the user (role) cannot be removed."

(Yes, that's passive voice, but it's correct, and avoids introducing a dangling "it", as in "fails when it cannot remove the user".)